---
id: 60a3c8f2-a101-4aed-b363-6344285345bb
title: "Alpha: Plan/Ideen"
tags: []
favorite: false
created_at: "2026-10-03T13:37:54.828Z"
updated_at: "2026-10-03T13:37:54.834Z"
---

![Ein Foto](../_attachments/foto.png)

[Bericht](../_attachments/bericht.pdf)
