---
id: 145dfc5d-22f9-4484-9393-8cd490cc365f
title: "Alpha: Plan/Ideen"
tags: []
favorite: false
created_at: "2026-10-07T21:32:39.556Z"
updated_at: "2026-10-07T21:32:39.560Z"
---

![Ein Foto](../_attachments/foto.png)

[Bericht](../_attachments/bericht.pdf)
